### PR TITLE
New branch

### PR DESCRIPTION
**Pull request by nxn404-n**
### In this branch

1. There is a contact me form.
2. The form works.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -184,5 +184,16 @@ <h3 class="about-header">About<br>Myself</h3>
                 <img src="req/sep.png" alt="#">
             </div>
     </section>
+    <section class="form-container">
+        <img src="req/contact form shapes-mobile.png" class="form-container__img">
+        <h3 class="contact-header">Contact me</h3>
+            <p class="contact-p">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form action="https://formspree.io/f/mayznbvq" method="post" class="form-container__form">
+            <input type="text" name="username" placeholder="Please insert your name" class="box-input" required>
```
Comment (binhussen) [inline index.html]: - Kindly, limit the name field input length to  `maxlength="30"` to satisfy the [requirment](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md).👌👍

File: index.html [hunk 2]
```diff
@@ -184,5 +184,16 @@ <h3 class="about-header">About<br>Myself</h3>
...
+            <input type="email" name="email" placeholder="Please insert your email" class="box-input" required>
+            <textarea name="comment" placeholder="Write your messege here" required></textarea>
```
Comment (binhussen) [inline index.html]: - Kindly, limit the message field input length to  `maxlength="500"` to satisfy the [requirment](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md).👌👍

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @nxn404-n , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @nxn404-n 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 - All linter checks passed ✔️
 - Descriptive pull request ✔️
 - Used GitHub flow ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**